renovate.json

```diff
@@ -82,6 +82,11 @@
         "@mui/system",
         "@mui/utils"
       ]
+    },
+    {
+      "groupName": "monorepo",
+      "matchPackageNames": ["@material-ui/monorepo"],
+      "matchUpdateTypes": ["lockFileMaintenance"]
```

Comment:
I tested in https://github.com/m4theushw/renovate-playground/blob/main/renovate.json and it didn't work. If `lockFileMaintenance` is not enabled, setting this option to this value doesn't have effect.

Reply:
Alright you seem to have this under better hands than mine, I'm giving up 😁